vocode/streaming/output_device/websocket_output_device.py

```diff
@@ -41,32 +34,26 @@ def mark_closed(self):
     async def process(self):
         while self.active:
             message = await self.queue.get()
-            await self.ws.send_text(message)
+            await self.ws.send_text(message.json())
 
-    def consume_nonblocking(self, chunk: bytes):
+    async def consume_nonblocking(self, chunk: bytes):
```

Comment:
why do we need to make this async?

Reply:
It's async in all the implementation we use